backend/.env.fuzz.example

```
@@ -0,0 +1,24 @@
+DJANGO_ALGOLIA_APPLICATION_ID=None
+DJANGO_ALGOLIA_EXCLUDED_LOCAL_INDEX_NAMES=None
+DJANGO_ALGOLIA_WRITE_API_KEY=None
+DJANGO_ALLOWED_HOSTS=*
+DJANGO_AWS_ACCESS_KEY_ID=None
+DJANGO_AWS_SECRET_ACCESS_KEY=None
+DJANGO_SETTINGS_MODULE=settings.fuzz
+DJANGO_CONFIGURATION=Fuzz
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DJANGO_CONFIGURATION key should go before the DJANGO_SETTINGS_MODULE key

(UnorderedKey)